kubejs/server_scripts/enigmatica/kubejs/constants/materials.js

```diff
@@ -173,15 +173,15 @@ const oreProcessingSecondaries = {
         createProcessingTime: 400
     },
     cloggrum: {
-        secondary: 'lead',
+        secondary: 'froststeel',
         createProcessingTime: 300
     },
     froststeel: {
-        secondary: 'cobalt',
+        secondary: 'utherium',
```

Comment:
This one we might want to keep as cobalt since cobalt is used to alloy it in expert. So it's a bit like natural electrum veins where you'll find some elemental gold and silver still.

Reply:
Please change this one back, we'll edit the Materialis compat :)